spec.html

```diff
@@ -288,7 +288,7 @@ <h1>%TypedArray%.prototype.toSpliced ( _start_, _deleteCount_, ..._items_ )</h1>
                         1. Repeat, while _k_ &lt; _actualStart_,
                             1. Let _Pk_ be ! ToString(𝔽(_k_)).
                             1. Let _kValue_ be ! Get(_src_, _Pk_).
-                            1. Perform ? Set(_target_, _Pk_, _kValue_, *true*).
+                            1. Perform ! Set(_target_, _Pk_, _kValue_, *true*).
                             1. Set _k_ to _k_ + 1.
                         1. For each element _E_ of _items_, do
                             1. Let _Pk_ be ! ToString(𝔽(_k_)).
```

Comment:
the `Set` on the next line here could still throw as the list of `_items_` to insert have not been inspected, they could be values that can't be inserted into a TypedArray.